### DISCUSSION TITLE
Resizing parent control (WinForm) causes black margins around ChromiumWebBrowser

### DISCUSSION THREAD

**Discussion by ILIA-BROUDNO** [cefsharp/CefSharp#4289] [Q&A] [answered]
I am using version 106 the WinForm version.
It does not matter if I place the browser control inside the form or the panel in the form.
When I resize the form I see large black margins on the left and top of the browser control.
The size of margins seem different every time and once they are there the buttons and other inputs in the browser no longer respond when the mouse is on top of them but respond when it is where they would be if the browser was drawn at the top left without the margins.
I tried changing dockinng, location etc - no dice.
Also from the mouse response I am pretty sure this is not an issue with my layout, but with control's rendering.
Assuming it's a bug, is there a work around?

**Comment by aimcosoftware** [comment]
Have you tried turning off GPU?

**Reply by ILIA-BROUDNO** [reply]
Yes with 
var settings = new CefSettings();
            settings.CefCommandLineArgs.Add("disable-gpu"); // Disable GPU acceleration
            settings.CefCommandLineArgs.Add("disable-gpu-vsync"); //Disable GPU vsync
No dice

**Reply by amaitland** [reply]
You can disable the GPU compositor in situations where making your application DPI aware is impractical. 

https://github.com/cefsharp/CefSharp/wiki/General-Usage#high-dpi-additional-info

**Comment by amaitland** [comment, marked as answer]
Sounds like you need to make your application DPI aware. See https://github.com/cefsharp/CefSharp/wiki/General-Usage#high-dpi-displayssupport

**Reply by ILIA-BROUDNO** [reply]
Added app.manifest to the project, uncommented this:
```
  <application xmlns="urn:schemas-microsoft-com:asm.v3">
    <windowsSettings>
      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
    </windowsSettings>
  </application>
```
And good bye bug
